src/Dialogs/Imap/ImapLoginPage.vala

```diff
@@ -68,7 +68,7 @@ public class OnlineAccounts.ImapLoginPage : Gtk.Grid {
             hexpand = true
         };
 
-        var display_name_hint_label = new Gtk.Label (_("The above name will be used to identify this account. Use for example “Work” or “Personal”.")) {
+        var display_name_hint_label = new Gtk.Label (_("Identify this account as for example “Work”, or “Personal” above.")) {
```

Comment:
This is not good English imo and is not clear for translation. I think the label is (or should be) closely enough associated with the entry to not require the word "above" anyway (there is nothing below).  No comma required before "or".

Possible alternative: `Identify this account, for example "Work" or "Personal"`

Reply:
```suggestion
        var display_name_hint_label = new Gtk.Label (_("Pick a name like “Work” or “Personal” for the account.")) {
```